features/support/recaptcha_helper.rb

```diff
@@ -1,45 +1,9 @@
-module RecaptchaHelper
-  module VerifyWithUnstub
```

Comment:
not using this anymore? Part of the new recaptcha version or something?

Reply:
Well that is old old old code, and I do not really see the point of doing that. 

![image](https://user-images.githubusercontent.com/64276/55074778-7bdbe780-5091-11e9-9da7-0aa8fbd086c9.png)

Reply:
Looks a way to overpass the verification give an environment, most probably for dev and testing purposes in the past

Reply:
Yes but `After` and `Before` does it better. And this is bad because it monkey patches code implementation in tests .. So you need to know the implementation to write your tests ...

Which means if you change the implementation but keep the behaviour your test will fail ... and it is really really really bad